src/core/trulens/experimental/otel_tracing/core/exporter.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+import logging
+from typing import Sequence
+
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter
+from opentelemetry.sdk.trace.export import SpanExportResult
+from trulens.core.database import connector as core_connector
+from trulens.core.schema import event as event_schema
+
+logger = logging.getLogger(__name__)
+
+
+class TruLensDBSpanExporter(SpanExporter):
+    """
+    Implementation of :class:`SpanExporter` that flushes the spans to the database in the TruLens session.
+    """
+
+    connector: core_connector.DBConnector
+
+    def __init__(self, connector: core_connector.DBConnector):
+        self.connector = connector
+
+    def export(self, spans: Sequence[ReadableSpan]) -> SpanExportResult:
+        try:
+            for span in spans:
+                context = span.get_span_context()
+
+                if context is None:
+                    logger.error(
+                        "Error exporting spans to the database: Span context is None"
+                    )
+                    return SpanExportResult.FAILURE
+
+                event = event_schema.Event(
+                    event_id=str(context.span_id),
+                    record=span.attributes,
+                    record_attributes=span.attributes,
```

Comment:
why are `record` and `record_attributes` the same? Isn't the former more fundamental stuff and the latter not as per https://docs.snowflake.com/en/developer-guide/logging-tracing/event-table-columns#label-event-table-schema?